### DISCUSSION TITLE
💡 Chromecast Support

### DISCUSSION THREAD

**Discussion by juanqp07** [DonutWare/Fladder#91] [Ideas]
### Summary

Add Chromecast support to be able to share movies on TVs more easily.

### Basic Example

You click the button available (for example, at the top) and it shows you all compatible devices to share the video and display it on the TV.

### Drawbacks

If the device on which the video is shared is not very powerful, the quality may be poor or it may have interruptions since it is not the same as a monitor or a mobile/tablet screen.

**Comment by PartyDonut** [comment]
This was a big one for me as well. It's something I looked at when starting this project. 

Biggest thing holding me back was no proper way (not that I could find) of support all platforms. In this case web was the one holding it back.

I've added it to Todo because I would still like to add chromecast support

**Reply by tbleiker** [reply]
Would it be a viable option for you to first implement Chromecast support for mobile clients (by using e.g. https://pub.dev/packages/flutter_chrome_cast)?

**Comment by juanqp07** [comment]
Add it without rushing, I really like the app on my mobile, but for Chromecast I use the native jellyfin one, it does its job but it gives me very poor performance.

**Comment by krishne35** [comment]
Any update on android tv support?
